iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -325,6 +334,14 @@ def _adjust_operator_bundle(
             log.info('Resolving image pull specs')
             bundle_metadata = _get_bundle_metadata(operator_manifest, pinned_by_iib)
             _resolve_image_pull_specs(bundle_metadata, labels, pinned_by_iib)
+        elif customization_type == 'perform_bundle_replacements':
+            log.info('Performing bundle replacements')
+            bundle_metadata = _get_bundle_metadata(operator_manifest, pinned_by_iib)
+            replacement_pullspecs = {}
+            for old, new in bundle_replacements.items():
+                if _is_bundle_image(old):
+                    replacement_pullspecs[ImageName.parse(old)] = ImageName.parse(new)
+            _replace_csv_pullspecs(bundle_metadata, replacement_pullspecs)
```

Comment:
Do we know what happens if the pullspec you are trying to replace is not present in the CSV? Will it throw an error?

Reply:
It won't throw an error. This is actually already tested with the test case I included --

```python
    bundle_replacements = {
        'quay.io/operator/image@sha256:654321': 'quay.io/operator/image@sha256:123456',
        'registry.access.company.com/operator/image@sha256:765432': 'foo.baz/image@sha256:234567',
    }
```
The first pullspec is present, the second isn't (at the time of doing the replacements).